Console/Command/NostoAccountRemoveCommand.php

```diff
@@ -0,0 +1,210 @@
+     * @param NostoAccountHelper $nostoAccountHelper
+     * @param NostoScopeHelper $nostoScopeHelper
+     * @param WriterInterface $appConfig
+     */
+    public function __construct(
+        NostoAccountHelper $nostoAccountHelper,
+        NostoScopeHelper $nostoScopeHelper,
+        WriterInterface $appConfig,
+        TypeListInterface $cacheTypeList
+    )
+    {
+        $this->nostoAccountHelper = $nostoAccountHelper;
+        $this->nostoScopeHelper  = $nostoScopeHelper;
+        $this->config = $appConfig;
+        $this->cacheTypeList =$cacheTypeList;
+        parent::__construct();
+    }
+
+    /**
+     * Configure the command and the arguments
+     */
+    public function configure()
+    {
+        $this->setName('nosto:account:remove')
+            ->setDescription('Remove Nosto Account Via CLI')
+           ->addOption(
+                self::SCOPE_CODE,
+                null,
+                InputOption::VALUE_REQUIRED,
+                'Store view code'
+            );
+        parent::configure();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function execute(InputInterface $input, OutputInterface $output)
+    {
+        $this->isInteractive = !$input->getOption('no-interaction');
+        $io = new SymfonyStyle($input, $output);
+        $scopeCode = $input->getOption(self::SCOPE_CODE) ?:
+            $io->ask('Enter Store Scope Code');
+
+        if ($this->removeNostoAccount($io, $scopeCode)) {
+            $io->success('Nosto account removed successfully');
+        } else {
+            $io->error('Could not complete operation');
+        }
+    }
+
+    /**
+     * Removes the local installation for the specific store
+     *
+     * @param SymfonyStyle $io
+     * @param $scopeCode
+     * @return bool
+     */
+    private function removeNostoAccount(SymfonyStyle $io, $scopeCode)
+    {
+        $store = $this->getStoreByCode($scopeCode);
+        if (!$store) {
+            $io->error('Store not found. Check your input.');
+            return false;
+        }
+        if (!$this->nostoAccountHelper->nostoInstalledAndEnabled($store)) {
+            $io->error('Store is not connected with any Nosto account.');
+            return false;
+        }
+        // If the script is non-interactive, do not ask for confirmation
+        $confirmOverride = $this->isInteractive ?
+            $confirmOverride = $io->confirm(
+                'Local Nosto account found for this store view. Remove account?',
+                false
+            ):
+            true;
+        if ($confirmOverride) {
+            $this->deleteAccount($store);
+            $this->cacheTypeList->cleanType('config');
+            return true;
+        } else {
+            $io->error('Removal was cancelled');
+            return false;
+        }
+    }
+
+    /**
+     * @param $scopeCode the storeview code
+     * @return \Magento\Store\Model\Store
+     */
+    private function getStoreByCode($scopeCode)
```

Comment:
I just realised that this is a carbon copy of `NostoAccountConnectCommand::getStoreByCode()`. One small refactoring thing would be to move this function from both `NostoAccountConnectCommand` and `NostoAccountRemoveCommand` to `Nosto\Tagging\Helper\Scope`, so we avoid duplication.